lib/beacon_api/controllers/v1/beacon_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule BeaconApi.V1.BeaconController do
+  use BeaconApi, :controller
+
+  @spec get_state_root(Plug.Conn.t(), any) :: Plug.Conn.t()
+  def get_state_root(conn, params) do
+    json(conn, %{
+      id: Map.get(params, "state_id"),
+      root: "0xbe41d3394dc8c461ab7049dbedb8944613ada8bd1743e091948f4d7b5ca8af36"
+    })
+  end
```

Comment:
I don't think this is the format the response expects. The example in the beacon API swagger is:

```json
{
  "execution_optimistic": false,
  "finalized": false,
  "data": {
    "root": "0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2"
  }
}
```

It's true that we don't have much of that info available yet, but even if we're hardcoding an answer, we should make the format compliant. That is, for example:

```elixir
json(conn, %{
  execution_optimistic: true,
  finalized: false,
  data: %{
    root: "0xbe41d3394dc8c461ab7049dbedb8944613ada8bd1743e091948f4d7b5ca8af36"
  }
})
```

Reply:
Modified